data/dbus/meson.build

```diff
@@ -0,0 +1,15 @@
+configure_file(
+  input: 'org.gnome.GPaste.Ui.dbus.in',
```

Comment:
any way to make this more "automatic", like looping over `*.dbus.in` and using the value for input and `s/dbus.in/service/` for output?

Reply:
We _could_ use a for loop here, sure. I personally prefer simplicity over trying to be smart in build systems though, but you choose.

Reply:
The more I use build tools such as cargo for rust, or most of the java build tools, the more I like it when all i need to do is drop a new file, to add a new feature, instead of having to explicitely reference it. Git is already there for that purpose.

Reply:
From what I know it's not possible to directly loop over files with meson. The best we could do is loop over an array of file names - but you'd have to specify that either way.